src/styles/typography.scss

```diff
@@ -10,11 +10,16 @@
   --text-size-5: calc(var(--text-size-4) * var(--scale));    // 31px
   --text-size-6: calc(var(--text-size-5) * var(--scale));    // 39px
   --text-size-7: calc(var(--text-size-6) * var(--scale));    // 48px
-  --text-size-8: calc(var(--text-size-7) * var(--scale));    // 61px
+  --text-size-8: calc(var(--text-size-7) * var(--scale));    // 56px
```

Comment:
This should be 61px, not 56px.
😮

![CleanShot 2022-01-17 at 12 54 43](https://user-images.githubusercontent.com/381395/149801744-e7e7b394-cb7f-4011-978c-b6f3d02e7753.png)

Reply:
True!